hieradata_aws/class/draft_cache.yaml

```diff
@@ -35,9 +35,5 @@ router::nginx::app_specific_static_asset_routes:
 
 router::nginx::check_requests_warning: '@0'
 router::nginx::check_requests_critical: '@0'
-router::nginx::proxy_busy_buffers_size: '16k'
-router::nginx::proxy_buffers: '8 8k'
-router::nginx::proxy_buffer_size: '8k'
```

Comment:
I’m assuming you’ve removed this because it’s expected it won’t be needed with the below changes. Unfortunately that isn’t the case because this is for a different nginx configuration (router). This also means that we wouldn’t be achieving Ian’s initial goal of increasing the buffer size for live router.

to configure router I’d suggest changing the proxy buffer configs on the puppet class: https://github.com/alphagov/govuk-puppet/blob/main/modules/router/manifests/nginx.pp#L79

If you set them to be the same as below you can still continue to remove this config as it won’t need overwriting.